frontend/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -48,6 +48,13 @@ const SingleToolTip: Story<TooltipProps> = (args) => {
   )
 }
 
+export const OpenTooltip = SingleToolTip.bind({})
```

Comment:
nit: should this story use the `Template` tooltip story? So we can show all 4 directions in a single story